daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -422,3 +422,13 @@ object Node {
     type WithTxValue[+Nid, +Cid] = F[Nid, Cid, Transaction.Value[Cid]]
   }
 }
+
+/** The constructor is private so that we make sure that only this object constructs
```

Comment:
This comment is misleading as there is no `private` keyword used below.

Reply:
:+1: